tutorials/quantum_simulation/Shadow estimation for VQS.ipynb

```diff
@@ -1,20 +1,5 @@
 {
```

Comment:
<p>add plt.show() to get rid of the <code>&lt;matplotlib.lines.Line2D at 0x7f79857e1d30&gt;</code>  line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>